gammapy/irf/background.py

```diff
@@ -130,6 +133,231 @@ def to_table(self):
+            data=table[bkg_name].data[0] * data_unit,
+            meta=table.meta,
+        )
+
+    @classmethod
+    def from_hdulist(cls, hdulist, hdu='BACKGROUND'):
+        """Create from `~astropy.io.fits.HDUList`."""
+        fits_table = hdulist[hdu]
+        table = fits_table_to_table(fits_table)
+        return cls.from_table(table)
+
+    @classmethod
+    def read(cls, filename, hdu='BACKGROUND'):
+        """Read from file."""
+        filename = make_path(filename)
+        hdulist = fits.open(str(filename))
+        return cls.from_hdulist(hdulist, hdu=hdu)
+
+    def to_table(self):
+        """Convert to `~astropy.table.Table`."""
+        meta = self.meta.copy()
+        table = Table(meta=meta)
+
+        table['THETA_LO'] = self.data.axis('offset').lo[np.newaxis]
+        table['THETA_HI'] = self.data.axis('offset').hi[np.newaxis]
+        table['ENERG_LO'] = self.data.axis('energy').lo[np.newaxis]
+        table['ENERG_HI'] = self.data.axis('energy').hi[np.newaxis]
+        table['BKG'] = self.data.data[np.newaxis]
+        return table
+
+    def to_hdulist(self, name='BACKGROUND'):
         """Convert to `~astropy.io.fits.BinTable`."""
-        return table_to_fits_table(self.to_table(), name)
\ No newline at end of file
+        hdu = table_to_fits_table(self.to_table(), name)
+        hdulist = fits.HDUList([fits.PrimaryHDU(), hdu])
+        return hdulist
+
+    def write(self, filename, name='BACKGROUND', **kwargs):
+        """Convert to `~astropy.io.fits.BinTable`.
+        Parameters
+        ----------
+        filename : str
+            File name
+        """
+        hdulist = self.to_hdulist(name=name)
+        hdulist.writeto(filename, **kwargs)
+
+    def evaluate(self, fov_offset, fov_phi=None, energy_reco=None, method=None, **kwargs):
+        """
+        Evaluate the `Background2D` at a given offset and energy.
+
+        Parameters
+        ----------
+        fov_offset : `~astropy.coordinates.Angle`
+            offset in the FOV
+        fov_phi: `~astropy.coordinates.Angle`
+            azimuth angle in the FOV. Not used for this class since the background model is radially symmetric
+        energy_reco : `~astropy.units.Quantity`
+            Vector of energy (1D) on which the model is evaluated
+        method : str {'linear', 'nearest'}, optional
+            Interpolation method
+        kwargs : dict
+            option for interpolation for `~scipy.interpolate.RegularGridInterpolator`
+
+        Returns
+        -------
+        array : `~astropy.units.Quantity`
+            Interpolated values, axis order is the same as for the NDData array
+
+        """
+        if energy_reco is None:
+            energy_reco = self.data.axis('energy').nodes
+
+        array = self.data.evaluate(offset=fov_offset, energy=energy_reco, method=method, **kwargs)
+        return array
+
+    def integrate_on_energy_range(self, tab_energy_band, energy_bins, fov_offset, fov_phi=None, method=None, **kwargs):
```

Comment:
@cdeil @registerrier 
For the moment, this method is only implemented in this 2D class since the issue on the NDataArray doesn't occur since we have only one axis for the spatial coordinates. For the moment since the NDataArray return a meshgrid of setx,dety, this is not possinle to implement it for the `Background3D class`